doc/Requirements Specification/Parts/Application.lyx

```diff
@@ -272,7 +272,7 @@ gls{PCM}
 
 \end_inset
 
-), which can hence be fixed in time.
+), which can then be fixed early on.
```

Comment:
How about changing it to "This leads to fast detection of arising problems (like implementation errors or unrealistic modelling in the PCM), which can then be fixed early on."? "fast" work well with "arising" and we'd avoid using "early" twice.